support/sier-codec/src/json.rs

```diff
@@ -66,3 +66,28 @@ pub fn transform_serde_obj<'a>(
 
     Ok(Object::new(def.as_ref(), values))
 }
+
+fn transform_sier_value<'a>(sier_value: &'a Value) -> Result<serde_json::Value, Error<'a>> {
+    Ok(match sier_value {
+        Value::Bool(b) => serde_json::Value::Bool(*b),
```

Comment:
Since this file is so JSON oriented, you can import serde_json types for use in the file. I think we already have `Value` as `SerdeValue` for example.